.pylintrc

```diff
@@ -516,7 +516,7 @@ valid-metaclass-classmethod-first-arg=cls
 max-args=5
 
 # Maximum number of attributes for a class (see R0902).
-max-attributes=7
+max-attributes=12
```

Comment:
can one just disable such nonsense stuff?

Reply:
wouldn't call it nonsense, but the specific settings maybe subject to discussion.